packages/orchestration/src/typeGuards.js

```diff
@@ -36,6 +36,15 @@ export const CosmosChainAddressShape = {
   value: M.string(),
 };
 harden(CosmosChainAddressShape);
+
+/** @type {TypedPattern<Caip10Record>} */
+export const Caip10RecordShape = {
+  namespace: M.string(),
+  reference: M.string(),
+  accountAddress: M.string(),
```

Comment:
Where do we plan to use this? I thought we were going to prefer `AccountId: ${string}:${string}:${string}` going forward, and Orchestration interfaces would accept `CosmosChainAddress | AccountId`?

Reply:
`parseAccountId` and `parseAccountIdArg` in `address.js` return this form, which is quite useful when you have a Caip10Address, and want the pieces.